src/components/students.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function Students() {
```

Comment:
When I move to another page, the nav button disappears and I cannot go back to previous page or go to other pages.
<img width="985" alt="image" src="https://github.com/daniel-august-mentorship/Demo-School/assets/6497134/26ea9279-ccf7-4498-8c65-1b4f6243b2ca">

> Note: You should be able to fix this without duplicating code in every page.